pkg/env/var.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2023 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package env
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+// Var represents an environment value. When loaded, it will expand from the current env
+type Var struct {
```

Comment:
```suggestion
type Env struct {
```
maybe redundant sometime but I think it express better the content that will store. Using `Var` is like this struct will hold only a value instead a key value pair

Reply:
for me an env var is always a key value set, calling it `Env` would result in `env.Env`, not clear to me. Perhaps we could rename it `env.EnvVar` but it sounds redundant, that'a why I ended up calling `env.Var`

Reply:
I think `Env` has a broader scope. I agree with Andrea. `Var` is already within a package called `env` which gives it the context you want to provide. This will be always called with `env.` prefix